src/main/java/com/alkemy/ong/application/exception/DefaultExceptionHandler.java

```diff
@@ -88,7 +88,16 @@ private ErrorResponse buildErrorResponse(HttpStatus httpStatus, String message,
         "The server could not complete the user registration because "
             + "the email address entered is already in use.");
     return new ResponseEntity<>(errorResponse, HttpStatus.CONFLICT);
+  }
 
+  @ExceptionHandler(value = UploadImageException.class)
+  public ResponseEntity<ErrorResponse> handleExternalServiceException(
+      UploadImageException e) {
+    ErrorResponse errorResponse = buildErrorResponse(
+        HttpStatus.INTERNAL_SERVER_ERROR,
+        e.getMessage(),
+        "The server could not complete the file upload");
```

Comment:
"The server could not complete the file upload" deberia ser el message y el mensaje de la exception deberia estar en el moreInfo.